sim/warrior/talents.go

```diff
@@ -9,10 +9,10 @@ import (
 )
 
 func (warrior *Warrior) ApplyTalents() {
-	warrior.AddStat(stats.Parry, core.ParryRatingPerParryChance*1*float64(warrior.Talents.Deflection))
 	warrior.AddStat(stats.MeleeCrit, core.CritRatingPerCritChance*1*float64(warrior.Talents.Cruelty))
 	warrior.AddStat(stats.MeleeHit, core.MeleeHitRatingPerHitChance*1*float64(warrior.Talents.Precision))
-	warrior.AddStat(stats.Dodge, core.DodgeRatingPerDodgeChance*1*float64(warrior.Talents.Anticipation))
+	warrior.PseudoStats.BaseDodge += 0.01*float64(warrior.Talents.Anticipation)
```

Comment:
Let's move these lines down 1 more so that all the `warrior.AddStat` calls are kept together.